eth/vm/base.py

```diff
@@ -111,8 +115,9 @@ def logger(self) -> logging.Logger:
     @abstractmethod
     def apply_transaction(self,
                           header: BlockHeader,
-                          transaction: BaseTransaction
-                          ) -> Tuple[BlockHeader, Receipt, BaseComputation]:
+                          transaction: BaseTransaction,
+                          *,
```

Comment:
Most of my python experience is with 2.7. I'm still a little bitter about `reduce` and [3113](https://www.python.org/dev/peps/pep-3113/) but wow, this is a nice feature.